Sprint-1/implement/sum.test.js

```diff
@@ -34,3 +33,23 @@ test.todo("given an empty array, returns 0")
 // Given an array with only non-number values
 // When passed to the sum function
 // Then it should return the least surprising value given how it behaves for all other inputs
+describe("sum", () => {
+    test("Given an empty array,it should return 0", () =>{
+        expect(sum([])).toEqual(0)
+    });
+    test("Given an array with just one number,it should return that number", () =>{
+        expect(sum([20])).toEqual(20)
+    });
+    test("Given an array containing negative numbers,it should still return the correct total sum", () =>{
+        expect(sum([-6,-1,3])).toEqual(-4)
+    });
+    test("Given an array with decimal/float numbers,it should return the correct total sum", () =>{
+        expect(sum([3.1, 2.1, 5])).toEqual(10.2)
```

Comment:
Decimal numbers in most programming languages (including JS) are internally represented in "floating point number" format. Floating point arithmetic is not exact. For example, the result of `46.5678 - 46 === 0.5678` is false because `46.5678 - 46` only yield a value that is very close to `0.5678`.

So `expect(sum([1.1, 1.1, 1.1])).toEqual(3.3);` would probably fail.

Can you find a more appropriate way to test a value (that involves decimal number calculations) for "equality?